website/next.config.js

```diff
@@ -16,24 +24,55 @@ const env = {
 
 const withNextra = nextra({
   theme: '@graphprotocol/nextra-theme',
-  staticImage: true,
-  flexsearch: false,
+  search: false,
   codeHighlight: false,
   defaultShowCopyCode: false,
   readingTime: true,
-  transform(result, { route }) {
-    if (route && !result.includes('getStaticProps')) {
-      const banner = `
-import { getPageMap } from '@/src/getPageMap'
+  transformPageMap(pageMap) {
+    const locale = pageMap[0].data.slice(0, 2)
+    return [
+      ...pageMap,
+      {
+        route: `/${locale}`,
+        name: 'index',
+        frontMatter: {
+          title: translate(translations, locale, 'index.title'),
+        },
+      },
+    ]
+  },
```

Comment:
I'm not quite sure I follow this. It seems to add a route for the `[locale]/index.mdx` file? Why is it needed for that particular route?

Reply:
it's a dynamic route, this is needed to translate `sidebarTitle`, both works `title` or `sidebarTitle` here but I'll rename to `sidebarTitle` to better understand